src/ontology/reports/gard-mondo-mapped-obsoletes.tsv

```diff
@@ -0,0 +1,254 @@
+?cls	?cls_label	?xref	?comment
```

Comment:
The output.

Example rows:
|?cls                                          |?cls_label                                                              |?xref     |?comment                                                                                                                                           |
|----------------------------------------------|------------------------------------------------------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------|
|<http://purl.obolibrary.org/obo/MONDO_0008331>|obsolete pseudopapilledema                                              |GARD:22114|This is a phenotypic feature and belongs in HPO.                                                                                                   |
|<http://purl.obolibrary.org/obo/MONDO_0009930>|obsolete pulmonary arteriovenous malformation                           |GARD:4584 |Reason: is a phenotype and not a disease. This is a trait or legacy term from OMIM and not suitabale for Mondo. Term to consider: HP:0006548       |
|<http://purl.obolibrary.org/obo/MONDO_0010330>|obsolete primary ciliary dyskinesia-retinitis pigmentosa syndrome       |GARD:17190|Reason: out of scope. Requested by ClinGen retinal experts to obsolete this term.                                                                  |
|<http://purl.obolibrary.org/obo/MONDO_0010582>|obsolete diabetes insipidus, neurohypophyseal type, X-linked inheritance|GARD:18603|The latest evidence for the existence of this form of this disease is from 1967, and was weak. From OMIM: \                                        |
|<http://purl.obolibrary.org/obo/MONDO_0011794>|obsolete Dravet syndrome                                                |GARD:10430|This is a distinct class from MONDO:0100079 epileptic encephalopathy, early infantile 6. See https://github.com/monarch-initiative/mondo/issues/745|

Reply:
I didn't see an "obsoletion reason" field. I thought maybe you meant exclusion reason, but that was not used either in this `mondo.owl`. Finally, I found that the reasons were contained within `rdfs:comment`.